xml/2018-10-14.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<email title="Weekly Digest">
+
+  <image
+    width="80"
+    src="https://i.imgur.com/5rJTZsq.jpg"
+    alt="Photo of digestive biscuits on a plate"
+    title=""/>
+  <hr/>
+
+  <text>
+    Hi again! It's time for another digest.
+
+    In the past week, we held four events: the Mums and Dads meeting, Linux talk,
+    Sponsors' Exhibition and the Bloomberg Decentralised App Development workshop.
+    These events were an overwhelming success with a total attendee count over all
+    four events of 1000. Thanks to everyone who participated and a special thanks
```

Comment:
over all four events of 1000. The Sponsors' Exhibition in particular saw an incredible 500 members attending! It would be very appreciated if you could provide your feedback for the Sponsors' Exhibition through this (link)[https://goo.gl/forms/qLc6GyaaNDPcGVuA3] so that we can improve our main industrial event next year! A massive thanks to everyone...